search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -95,3 +105,88 @@
                        "concat humanized rows:" t3))))]
       (info "Create report " t4)
       (str string-writer))))
+
+(defn safe-generate-humanizers-report-csv
```

Comment:
We don't run this from the REPL, but the job scheduler runs in the dev-system, so this code actually does now get run in the dev-system and the REPL will see errors here:
```
user=> 2017-05-12 09:43:27.384 [HumanizerReportGeneratorJob] ERROR [cmr.search.services.humanizers.humanizer-report-service] - HumanizerReportGeneratorJob caught exception.
                                                      org.quartz.simpl.SimpleThreadPool$WorkerThread.run         SimpleThreadPool.java:  573
                                                                         org.quartz.core.JobRunShell.run              JobRunShell.java:  202
             cmr.search.services.humanizers.humanizer-report-service.HumanizerReportGeneratorJob/execute  humanizer_report_service.clj:  153
                  cmr.search.services.humanizers.humanizer-report-service.HumanizerReportGeneratorJob/fn  humanizer_report_service.clj:  156
cmr.search.services.humanizers.humanizer-report-service/safe-generate-humanizers-report-csv/invokeStatic  humanizer_report_service.clj:  110
     cmr.search.services.humanizers.humanizer-report-service/generate-humanizers-report-csv/invokeStatic  humanizer_report_service.clj:   88
                            cmr.search.services.humanizers.humanizer-service/get-humanizers/invokeStatic         humanizer_service.clj:   84
                   cmr.search.services.humanizers.humanizer-service/fetch-humanizer-concept/invokeStatic         humanizer_service.clj:   56
                                             cmr.common.services.errors/throw-service-error/invokeStatic                    errors.clj:   27
                                                                       clojure.core/ex-info/invokeStatic                      core.clj: 4617
clojure.lang.ExceptionInfo: Humanizer does not exist
```

This will go away if you run the humanizer tests, though.

How about a compromise? I'll add a check here that will throw if it's not the error above ...